src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form data-qa="big" class="form_field">
+      <label class="search-bar search-bar--big" >
+        <img 
+          src="./images/Search.svg" 
+          alt="search" 
+          class="search-icon search-icon--big"
+        >
+        <input 
```

Comment:
Try to change the order for `img` and `input` tags to prevent the icon from disappearing. If you have any questions ask them in the fe_chat, please
```
<input ...
<img ...

Reply:
done